tests/Support/SupportStringableTest.php

```diff
@@ -1179,4 +1179,13 @@ public function testToDateThrowsException()
 
         $this->stringable('not a date')->toDate();
     }
+
+    public function testArrayAccess()
+    {
+        $str = $this->stringable('my string');
+        $this->assertSame('m', $str[0]);
+        $this->assertSame('s', $str[4]);
```

Comment:
This should be `$this->assertSame('t', $str[4]);` or `$this->assertSame('s', $str[3]);`.

|  index |  char |
|:-------:|:-----:|
| 0 | m |
| 1 | y |
| 2 | SPACE |
| 3 | s |
| 4 | t |
| 5 | r |
| 6 | i |
| 7 | n |
| 8 | g |

Reply:
Indeed, thanks for the catch.